src/gui/speedlimitdialog.cpp

```diff
@@ -56,10 +56,10 @@ SpeedLimitDialog::SpeedLimitDialog(QWidget *parent)
 {
     m_ui->setupUi(this);
 
-    m_ui->labelGlobalSpeedIcon->setPixmap(Utils::Gui::scaledPixmap(UIThemeManager::instance()->getIcon(QLatin1String("slow_off"))
-                                                , this, style()->pixelMetric(QStyle::PM_SmallIconSize, nullptr, this)));
-    m_ui->labelAltGlobalSpeedIcon->setPixmap(Utils::Gui::scaledPixmap(UIThemeManager::instance()->getIcon(QLatin1String("slow"))
-                                                , this, style()->pixelMetric(QStyle::PM_SmallIconSize, nullptr, this)));
+    m_ui->labelGlobalSpeedIcon->setPixmap(Utils::Gui::scaledPixmapSvg(UIThemeManager::instance()->getIconPath(QLatin1String("slow_off"))
```

Comment:
@xavier2k6 
Note that I've made a specific commit for this `scaledPixmap` -> `scaledPixmapSvg` change in another PR (#16585) so this PR will incorporate only the necessary changes, I hope you are ok with it.

Reply:
>I hope you are ok with it.

No problem with me at all, I've been working/looking in to the redesigned icons PR & the `24` size is more than enough anyway with the proposed icons there.

I think though that your initial change request to just use `, this, 24));` was a good one for below:

https://github.com/qbittorrent/qBittorrent/blob/889d994dbc0d6c533bfa4ad28c7bb35b8678ad67/src/gui/speedlimitdialog.cpp#L60

https://github.com/qbittorrent/qBittorrent/blob/889d994dbc0d6c533bfa4ad28c7bb35b8678ad67/src/gui/speedlimitdialog.cpp#L62

I don't insist on the change & I'll leave it to you. 👍

Reply:
@Chocobo1 Closing this PR, thank you.